### PR TITLE
chore(jenkins): use project npmrc file

### PR DESCRIPTION
**Pull request by adamweeks**
This change will append and reset the project's npmrc file instead of the home directory's one which was causing issues on jenkins.

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview ready!

Built with commit 0312a5f15ed886656170658ecb257351d6499a1d

https://deploy-preview-604--hopeful-hamilton-4090b9.netlify.com

File: Jenkinsfile.jssdk [hunk 1]
```diff
@@ -24,13 +24,11 @@ ansiColor('xterm') {
           withCredentials([
             string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
           ]) {
-            sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > $HOME/.npmrc'
             sh '''#!/bin/bash -ex
             source ~/.nvm/nvm.sh
             nvm install v8.9.1
             nvm use v8.9.1
             npm install
-            rm -f $HOME/.npmrc
```
Comment (bzang) [inline Jenkinsfile.jssdk]: Do we no longer need private repo access for updating spark version? If not, I think we can safely remove the withCredentials call from this block.

Comment (adamweeks) [reply Jenkinsfile.jssdk]: I was going to leave it in until the test users module is fully public.

Comment (bzang) [reply Jenkinsfile.jssdk]: That makes sense, but this change seems to indicate that we're removing the line that injects the token into `.npmrc`, so the `NPM_TOKEN` env var isn't used anywhere.

Comment (adamweeks) [reply Jenkinsfile.jssdk]: Oh, I see, yea, we probably don't need it then.

**Pull request merged**